extmod/modbluetooth.c

```diff
@@ -1203,6 +1297,24 @@ void mp_bluetooth_gatts_on_indicate_complete(uint16_t conn_handle, uint16_t valu
     schedule_ringbuf(atomic_state);
 }
 
+
+bool mp_bluetooth_gatts_on_read_request(uint16_t conn_handle, uint16_t value_handle) {
```

Comment:
was there a reason to shift these 2 functions up to this position?

Reply:
Conceptually I was trying to group the peripheral/server functions together, and the central/client functions together. When I added these two (in earlier PRs) they accidentally went after the central functions. This makes them match the non-ringbuf order (which got this right).

Reply:
Ok!